client/src/components/ProjectWizard/RuleCalculation/RuleCalculation.js

```diff
@@ -147,6 +148,25 @@ const useStyles = createUseStyles({
     flexBasis: "50%",
     flexGrow: "1",
     flexShrink: "1"
+  },
+  tooltip: {
+    color: "rgb(30, 36, 63) !important",
+    minWidth: "200px",
+    maxWidth: "400px",
+    fontFamily: "Arial",
+    fontSize: 12,
+    lineHeight: "16px",
+    fontWeight: "bold",
+    "-webkit-box-shadow": "0px 0px 8px rgba(0, 46, 109, 0.2)",
```

Comment:
i don't think you need add the webkit and moz. is there a reason why you was these? i think most modern browsers support box shadow and definitely support border radius

Reply:
I'm assuming this may have been a copy pasta? but if there's a reason for adding them, definitely open to hearing why.